src/material/core/option/option.scss

```diff
@@ -53,11 +53,16 @@
     }
   }
 
-  // If the MDC list is loaded after the option, this gets overwritten which breaks the text
-  // alignment. Ideally we'd wrap all the MDC mixins above with this selector, but the increased
-  // specificity breaks some internal overrides.
   &.mdc-list-item {
+    // If the MDC list is loaded after the option, this gets overwritten which breaks the text
+    // alignment. Ideally we'd wrap all the MDC mixins above with this selector, but the increased
+    // specificity breaks some internal overrides.
     align-items: center;
+
+    // List items in MDC have a default background color which can be different from the container
+    // in which the option is projected. Set the base background to transparent since options
+    // should always have the same color as their container.
+    background: transparent;
```

Comment:
I think this is getting fixed by a new PR by Andrew
https://github.com/angular/components/pull/28504/files#diff-67483cc6b3bd3044567dc46603824ee7933a8ae08fd79d933f417aa1cb643fd0R475

Reply:
It's kinda related, although this is something we should've been doing in the option all along.

Reply:
Does this mean Andrew should close his PR in favor of this? I believe this would make Andrew's change obsolete, right?

Reply:
Nope, this is for `mat-option` specifically which happens to inherit some styles from the list. It's valid for a list to have a background.